w/index.md

```diff
@@ -0,0 +1,8 @@
+# List of the artists starting with W
```

Comment:
This looks good, except you don't need the 'the'. See the template - your title should match that.